tests/legacy-cli/e2e/tests/misc/module-resolution.ts

```diff
@@ -60,12 +68,12 @@ export default async function () {
       '@firebase/polyfill': ['@firebase/polyfill/index.ts'],
     };
   });
-  await expectToFail(() => ng('build'));
+  await ng('build');
```

Comment:
these shouldn't fail as typescript is able to resolve these even with wrong paths. Since they are under `node_modules`